Manuals/Tools/Altibase_trunk/eng/dataCompJ User's Manual.md

```diff
@@ -0,0 +1,1073 @@
+| Lower case words | Indicate program elements set by the user, such as table names, column names, file names, etc. | SELECT ename FROM employee;                                  |
+| Upper case words | Keywords and all elements provided by the system appear in upper case. | DESC SYSTEM_.SYS_INDICES_;                                   |
+
+#### Related Documentations
+
+For more detailed information, please refer to the following documents.
+
+- Installation Guide
+
+- Getting Started Guide
+
+- Administrator’s Manual
+
+- Replication Manual
+
+- Precompiler User’s Manual
+
+- API User’s Manual
+
+- Altibase C Interface Manual
+
+- iSQL User’s Manual
+
+- Utilities Manual
+
+- General Reference
+
+- Error Message Reference
+
+#### Altibase Welcomes Your Comments and Feedbacks
+
+Please let us know what you like or dislike about our manuals. To help us with better future versions of our manuals, please tell us if there is any corrections or classifications that you would find useful.
+
+Include the following information:
+
+- The name and version of the manual that you are using
+- Any comments about the manual
+- Your name, address, and phone number
+
+If you need immediate assistance regarding any errors, omissions, and other technical issues, please contact [Altibase's Support Portal](#http://altibase.com/support-center/en/).
+
+Thank you. We always welcome your feedbacks and suggestions.
+
+# 1. Introducing dataCompJ
+
+This chapter introduces dataCompJ and explains how to install it. This chapter consists of the following sections:
+
+- Overview
+
+- System Requirements
+
+- Installation and Uninstallation
+
+### Overview
+
+Altibase dataCompJ is a utility designed to guarantee data consistency between two heterogeneous databases and resolve data inconsistency if any. dataCompJ compares Altibase with another heterogeneous database on a table-by-table basis, and outputs information on any inconsistency it detects. Also, it offers an efficient feature for synchronizing two databases by applying the inconsistent data to slave database in the event of data inconsistency.
+
+Data managed by dataCompJ are replicated by Adapter for Oracle or Adapter for JDBC provided by Altibase.
+
+A proper use of Altibase dataCompJ would be a cornerstone of effective data management with advantageous usability along with its advanced technical performance.
+
+### System Requirements
+
+This section discusses the system requirements for dataCompJ and compatible database management systems.
+
+#### Hardware Requirements
+
+- CPU: Pentium III 800MHz or better
+- Main memory: Minimum 512MB,  4GB or more recommended
+- Disk: 50MB or more free space
+
+#### Software Requirements
+
+- Oracle, OpenJDK or IBM Java Runtime Environment 8 or higher
+
+dataCompJ is a pure Java-based client application relying on the JAVA Runtime Environment (JRE) instead of the client's hardware or an operating system.
```

Comment:
dataCompJ is a pure Java-based client application which can be run independent of  client's hardware or an operating system. However, since it relies on the JAVA Runtime Environment (JRE), the appropriate version of JAVA Runtime Environment (JRE) should be installed. In addition, JAVA_HOME environment variable which indicates the directory Java is installed should be specified.

반영합니다~